contracts/ColonyNetworkStaking.sol

```diff
@@ -0,0 +1,306 @@
+pragma solidity ^0.4.17;
+pragma experimental "v0.5.0";
+pragma experimental "ABIEncoderV2";
+
+import "../lib/dappsys/auth.sol";
+import "./Authority.sol";
+import "./IColony.sol";
+import "./EtherRouter.sol";
+import "./Token.sol";
+import "./ColonyNetworkStorage.sol";
+import "./IColonyNetwork.sol";
+
+
+contract ColonyNetworkStaking is ColonyNetworkStorage {
+
+  event Address(address _a);
+  event Bool(bool _b);
```

Comment:
Are these events left over from debugging? Neither is used.

Reply:
Yup, they are. Good catch.